subprojects/core/src/main/java/org/gradle/api/internal/tasks/userinput/DefaultUserInputHandler.java

```diff
@@ -98,15 +86,10 @@ public boolean askBooleanQuestion(String question, final boolean defaultValue) {
             String defaultString = defaultValue ? "yes" : "no";
             builder.append(defaultString);
             builder.append(") [");
-            builder.append(StringUtils.join(YES_NO_CHOICES, ", "));
+            builder.append(StringUtils.join(YesNoQuestionPromptEvent.YES_NO_CHOICES, ", "));
```

Comment:
❗ Shouldn't this be `BooleanQuestionPromptEvent.LENIENT_YES_NO_CHOICES`?

Reply:
I'm not sure it needs to list all of the variations of "yes" or "no". Maybe we could format this message in a better way, but this is a refactoring PR and doesn't attempt to change behavior from the user perspective.